### ISSUE TITLE
Installation failing while trying to fetch from kde flatpack repo

### ISSUE THREAD

**Issue by Tox46** [JerrySM64/Xwayland-Video-Bridge-Quick-Setup#1]
I'm trying to install using your script (can't find a instructions to install the video-bridge anywhere else :face_with_spiral_eyes:  ) and while i try to add the kde repo to flatpack it gives me this error: 
`error: Can't load uri https://distribute.kde.org/kdeapps.flatpakrepo: While fetching https://distribute.kde.org/kdeapps.flatpakrepo: [6] Couldn't resolve host name`
and everything falls apart from there.

The error explains well what's happening here, hope this get fixed quick.

**Comment by De-Clan-C** [comment]
As far as I can tell, the url doesn't exist. I found the [flapakrepo](https://github.com/KDE/flatpak-kde-applications/blob/master/kdeapps.flatpakrepo) file on the [KDE repo](https://github.com/KDE/flatpak-kde-applications) here on github, and even installing the repo through that doesn't work. So I'm guessing that maybe that site is an internal thing for the KDE project and only the devs have access to. But either way, we can't connect to it. The script itself has no issues.

**Comment by Tox46** [comment]
>  I found the [flapakrepo](https://github.com/KDE/flatpak-kde-applications/blob/master/kdeapps.flatpakrepo) file on the [KDE repo](https://github.com/KDE/flatpak-kde-applications) here on github

On the README.md of the repo you mentioned i found a link to some kind of kde/flatpak wiki, where you can find an installation tutorial. Funnily enough the first method to install KDE apps doesn't work, or at least you cant install xwaylandvideobridge... that you can install if you follow the tutorial for nightly apps.

this is the link to the wiki: https://userbase.kde.org/Tutorials/Flatpak

the shell code i run to install the kde repo and xwaylandvideobridge is this:
`flatpak install --user --or-update https://cdn.kde.org/flatpak/kde-runtime-nightly/org.kde.Platform.flatpakref
flatpak install --user --or-update https://cdn.kde.org/flatpak/xwaylandvideobridge-nightly/org.kde.xwaylandvideobridge.flatpakref`

i don't know if it is intended to have xwaylandvideobridge only on the nightly repo, once clarified this detail i'll make a pull request to this repo

**Comment by JerrySM64** [comment]
First of all, I'm sorry I didn't get around to look into the issue earlier. Have been busy the last days. The repo I added was the KDE nightly repo and back when I added it, it worked perfectly. The reason why I used nightly was due to the fact that the Xwayland Video Bridge was so new it only was available in the nightly one. Today it might not be needed anymore to get it from nightly anymore since Plasma 6 ships with the Bridge as far as I know.

I haven't done a Gentoo OpenRC install yet to check if the Flatpak installation works in the testing branch. That's why I haven't looked after the repo all those months. I will check that and if that still needs to come from nightly, I will check out the PR.

Thanks for bringing this to my attention!
